src/widgets/forms/form-view.component.tsx

```diff
@@ -23,8 +30,23 @@ const FormView: React.FC<FormViewProps> = ({
   patientUuid,
   encounterUuid
 }) => {
+  const searchTimeout = 300;
```

Comment:
Nit, and matter of opinion: unless this is used in multiple places, there's not much of a point to not just inline it. The additional declaration here at the top just adds complexity.